File: string/1662.check_if_two_string_arrays_are_equivalent.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution
{
public:
    bool arrayStringsAreEqual(vector<string> &word1, vector<string> &word2)
    {
        string st1, st2;
        for (auto ith : word1)
        {
            st1 += ith;
        }
        for (auto ith : word2)
        {
            st2 += ith;
        }

        return st1 == st2;
    }
};

int main()
{
}

File: string/1662.check_if_two_string_arrays_are_equivalent.py
class Solution:
    def arrayStringsAreEqual(self, word1: List[str], word2: List[str]) -> bool:
        st1, st2 = "", ""
        for ith in word1:
            st1 += ith
        for ith in word2:
            st2 += ith
        return st1 == st2
